src/App.js

```diff
@@ -1,30 +1,35 @@
-import React from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
-
-function App() {
-
-  const [todoList, setTodoList ] = React.useState([]);
+import React from "react";
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
+
+const useSemiPersistentState = () => {
+  const [todoList, setTodoList] = React.useState(
+    JSON.parse(localStorage.getItem("savedTodoList")) || [
+      { title: "React", id: Date.now() },
+    ]
+  );
```

Comment:
Good idea to set it equal to an array as an option, just in case the JSON doesn't parse correctly or there's nothing in local storage.